File: packages/cli/test/fixtures/ranking-corpora/python/rankingpy/handlers/update.py
"""Changes an existing user's display name."""

from rankingpy.util.logger import log_error, log_info


def update_user(user_id: int, name: str) -> None:
    if user_id <= 0:
        log_error("update_user", ValueError("id must be positive"))
        return
    log_info(f"updated user {name}")

File: packages/cli/test/fixtures/ranking-corpora/rust/src/update.rs
use crate::util;

/// Changes an existing user's display name, logging the outcome through the
/// shared `util` module.
pub fn update_user(id: u32, name: &str) -> Result<(), String> {
    if id == 0 {
        let err = "id must be positive".to_string();
        util::log_error("update_user", &err);
        return Err(err);
    }
    util::log_info(&format!("updated user {}", name));
    Ok(())
}
